templates/events-list.php

```diff
@@ -16,26 +21,29 @@
 		<?php
 		while ( $current_events_query->have_posts() ) :
 			$current_events_query->the_post();
-			$event_start = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format('l, F j, Y');
+			$event_start = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format( 'l, F j, Y' );
+			$event_url   = gp_url( wp_make_link_relative( get_the_permalink() ) );
 			?>
 			<li class="event-list-item">
 				<span class="event-list-date"><?php echo esc_html( $event_start ); ?></span>
-				<a href="<?php echo esc_url( gp_url( wp_make_link_relative( get_the_permalink() ) ) ) ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
+				<a href="<?php echo esc_url( $event_url ); ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
 				<p><?php the_excerpt(); ?></p>
 			</li>
-		<?php
+			<?php
 		endwhile;
 		?>
 	</ul>
 
 	<?php
-	echo paginate_links(
-		array(
-			'total'     => $current_events_query->max_num_pages,
-			'current'   => max( 1, $current_events_query->query_vars['current_events_paged'] ),
-			'format'	=> '?current_events_paged=%#%',
-			'prev_text' => '&laquo; Previous',
-			'next_text' => 'Next &raquo;',
+	echo wp_kses_post(
+		paginate_links(
+			array(
+				'total'     => $current_events_query->max_num_pages,
+				'current'   => max( 1, $current_events_query->query_vars['current_events_paged'] ),
+				'format'    => '?current_events_paged=%#%',
+				'prev_text' => '&laquo; Previous',
+				'next_text' => 'Next &raquo;',
+			)
```

Comment:
Could you add a `?? ''` to both `paginate_links()` inside this PR to avoid the error  Deprecated: preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /wp-includes/kses.php on line 1744`

Reply:
Oops, good catch. I just saw the deprecation message in the the events list page. Fixed.